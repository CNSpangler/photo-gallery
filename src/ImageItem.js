import React, { Component } from 'react';

export default class ImageItem extends Component {
    render() {
        return(
            <li>
                <h3>{this.props.imageTitle}</h3>
                <img src={this.props.imageSource} alt={this.props.imageDescription} />
                <h4>This creature has {this.props.imageHorns} horn(s)! 
                {this.props.imageDescription}</h4>
            </li>
        )
    }
};