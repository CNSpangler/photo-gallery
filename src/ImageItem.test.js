import React from 'react';
import renderer from 'react-test-renderer';
import ImageItem from './ImageItem.js';

it('renders correctly', () => {
    const tree = renderer
        .create(<ImageItem />)
        .toJSON();
        expect(tree).toMatchSnapshot();
});