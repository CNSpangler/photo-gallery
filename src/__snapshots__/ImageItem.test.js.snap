// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="li-div"
>
  <li>
    <h3 />
    <img />
    <h4>
      This creature has 
       horn(s)!
    </h4>
  </li>
</div>
`;
